cmd/util.go

```diff
@@ -271,6 +275,12 @@ Alternatively, you can specify the version with --version
 `
 
 func getKubeVersion() (string, error) {
+
+	if k8sVer, err := getKubeVersionFromRESTAPI(); err == nil {
+		glog.V(2).Info(fmt.Sprintf("Kubernetes REST API Reported version: %s", k8sVer))
+		return k8sVer, nil
+	}
```

Comment:
This looks untested as it stands. Should we add coverage? Might be better to look at this after we've addressed https://github.com/aquasecurity/kube-bench/pull/518/files#r344931996

Reply:
The wrapping method that calls this method is difficult to test because there is a scenario that performs a search against the root (**'/'**) directory. 
I'm thinking about refactoring that function in the future, but I need to coordinate with Liz to minimize impact on the cilent.

Reply:
Fair point.